docs/general/thousand-validators.md

```diff
@@ -62,10 +119,9 @@ Thousand Validators programme. They are formatted like "`stash` / `controller`".
 
 A time delay proxy is used as the interaction method for some of these accounts.
 
-Since approximately early January 2021, the nominators will select an automatic number of validators
-to nominate based on the lowest amount staked for a validator and the amount of funds it holds. This
-can be anywhere from a few validators receiving nomination from a single 
-nominator to the max of 16.
+{{ kusama:Within the Kusama programme there are several other nominator accounts that can exhaustively be determined by parsing data found [here](https://kusama.w3f.community/nominators):kusama }}
+
+Since approximately early January 2021, the nominators will select an automatic number of validators to nominate based on the lowest amount staked for a validator and the amount of funds it holds. This can be anywhere from a few validators receiving nomination from a single nominator to the max of {{ kusama: 24:kusama }}{{ polkadot: 16 :polkadot }}.
```

Comment:
```suggestion
Since approximately early January 2021, the nominators will select an automatic number of validators to nominate based on the lowest amount staked for a validator and the amount of funds it holds. This can be anywhere from a few validators receiving nomination from a single nominator, to the max of {{ kusama: 24 :kusama }}{{ polkadot: 16 :polkadot }} nominators on {{ kusama: Kusama :kusama }}{{ polkadot: Polkadot :polkadot }}.
```